runtime/zlib/deflate.h

```diff
@@ -329,7 +329,6 @@ void ZLIB_INTERNAL _tr_stored_block OF((deflate_state *s, charf *buf,
     s->dyn_ltree[cc].Freq++; \
     flush = (s->sym_next == s->sym_end); \
    }
-/* Source is modified from the original zlib version so it will compile on Windows. */
```

Comment:
Are other changes marked with similar comments for AIX (e.g. in `crc32.c`) still required?

Reply:
Yes. I removed the comment here because the zlib 1.2.13 code has the same change we were adding here before.

Reply:
To put it another way, the AIX compiler we use for 32-bit hasn't changed, so the modifications for AIX are still required.